tests/conftest.py

```diff
@@ -202,6 +202,14 @@ def __gen_baselines(self, result_dict):
                         else:
                             value = round(function_node["value"])
 
+                        # It may happen that the results contain a kernel
+                        # version for which the config does not have an entry.
+                        # It could be that the S3 bucket contains an outdated
+                        # or new kernel version.
+                        assert kernel in baselines[measure],\
+                            "Results contain kernel version for which the" \
```

Comment:
let's also output the said kernel version

Reply:
No longer applies.